llama_stack/apis/common/type_system.py

```diff
@@ -54,6 +54,11 @@ class AgentTurnInputType(BaseModel):
     type: Literal["agent_turn_input"] = "agent_turn_input"
 
 
+class DialogType(BaseModel):
+    # expects List[Message] for messages
```

Comment:
I think adding a comment here which says that "this type semantically contains the output label whereas ChatCompletionInputType does not" would be good